docs/develop-resources/deep-dives/1-add-mockgcp-tests.md

```diff
@@ -29,6 +29,12 @@ Create a directory for your resource [pkg/test/resourcefixture/testdata/basic](p
 Just to add the `create.yaml`,  `update.yaml` and `dependencies.yaml` (if applicable) to each bottommost directory.
 These three files are all Kubernetes config files, except that `create.yaml` and `update.yaml` only holds the your Config Connector CR object, and the `dependencies.yaml` holds all the dependent objects.
 
+Note: Do not add your GCP Project as a dependency in `dependencies.yaml` (even with the `cnrm.cloud.google.com/deletion-policy: abandon` annotation). Adding your project as a dependency will cause Config Connector to add the label `cnrm-test: true` to your project during test runs, which likely not desirable. Instead, you can refer to your development project in the `create.yaml` and `update.yaml` resources via an external project reference, with the `${projectId}` template variable. For example:
```

Comment:
Suggest adding the condition that 1. for hack/record-gcp but not hack/compare-mock. 2. "your GCP project" refers to the default project in `gcloud config get project` 3. explain a little bit more on what happened to "cnrm-test: true" label.

Reply:
Fixed :)